app/main.py

```diff
@@ -1,6 +1,29 @@
+import os
+import requests
+
+
 def get_weather() -> None:
-    # write your code here
-    pass
+
+    api_key = os.environ.get("API_KEY")
+    city = "Paris"
+    url = ("https://api.weatherapi.com/v1/current.json?"
+           f"key={api_key}&q={city}")
```

Comment:
read the checklist. i think it is better to specify query params using `params=` argument

Reply:
Yes, and constants should be uppercased, f.e. BASE_URL, API_KEY, CITY